web/src/components/Presentation/AttributeList.vue

```diff
@@ -98,6 +98,14 @@ export default defineComponent({
       alternateIdentifiers,
       displayFields,
       relatedBiosamples,
+      images: {
+        // eslint-disable-next-line global-require
+        gold: require('@/assets/GOLD.png'),
+        // eslint-disable-next-line global-require
+        img: require('@/assets/IMG.png'),
+        // eslint-disable-next-line global-require
+        emsl: require('@/assets/EMSL.png'),
+      },
```

Comment:
This is great, it's exactly what we do for study links in [StudyPage.vue](https://github.com/microbiomedata/nmdc-server/blob/da8bb307b45492fe971f05c62d5d0d5ee7b60957/web/src/views/IndividualResults/StudyPage.vue#L222-L229).

I do however wonder about the use of `require` here. Most of the code base follows ES6 import/export syntax, but here we're using `require` to import images.

I think it might be worth opening up an issue to track this, and at some point investigate switching over to using `import` syntax. I don't think that needs to be done as part of this PR though.

Reply:
FYI I've opened issue #1579 to track updating these `require` statements in the future.